### PR TITLE
Added blog post for connected registry

### PR DESCRIPTION
**Pull request by toddysm**

### CODE REVIEW

File: docs/blog/connected-registry.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+description: Private preview for ACR connected registry feature.
+ms.topic: post
+ms.date: 01/05/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Private Preview - ACR Connected Registry Feature
+We are announcing the private preview of Azure Container Registry (ACR) connected registry feature.
```
Comment (jaysterp) [inline docs/blog/connected-registry.md:12]: @toddysm nit: "We are announcing the private preview of the Azure Container Registry (ACR) connected registry feature.". Added a "the" before Azure Container Registry. Alternately

File: docs/blog/connected-registry.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+The connected registry feature of ACR allows you to deploy a registry on your premises and synchronize images between the ACR and your premises. It brings the container images and OCI artifacts closer to your on-premise container workloads and increases their performance and reliability.
```
Comment (jaysterp) [inline docs/blog/connected-registry.md:14]: @toddysm curious, will we use "on-premises" or "on-premise" phrasing? I see IoT Edge uses "on-premises" https://docs.microsoft.com/en-us/azure/iot-edge/about-iot-edge?view=iotedge-2018-06

**Comment by sajayantony** [comment]
Unfortunately - Added some comments here - https://github.com/Azure/acr/pull/501#issuecomment-754814631
Nothing major just minor nits.

File: docs/blog/connected-registry.md (hunk 2 above)
Comment (SteveLasker) [reply docs/blog/connected-registry.md:14]: The wording around "your premises" and on-premises is a bit confusing: @dlepow for assistance here

File: docs/blog/connected-registry.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+ACR connected registry can be used in conjunction with Azure IoT Edge, Azure Arc, Azure Stack as well as other edge container workloads.
```
Comment (SteveLasker) [inline docs/blog/connected-registry.md:16]: Can we make IoT Edge and Azure Stack product links?
Is "other edge container" workloads the right terminology? I thought "edge" was verboten? @dlepow?

File: docs/blog/connected-registry.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Right now, the feature is scheduled for limited private preview with selected customers starting on January 15th 2021. If you want to be considered for the private preview, please submit your contact details using this [form](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR1OsLxas9SdIhfyFenqqkolUMkFKMTdDSU45SFQzU0o0WUNROVAySkRINy4u).
```
Comment (SteveLasker) [inline docs/blog/connected-registry.md:18]: remove "right now". 
Connected registry is currently in limited preview. To request private preview access, submit requests via this form...

We should avoid dates and refer to customers as "selected"...

Comment (mnltejaswini) [inline docs/blog/connected-registry.md:18]: Do you need to indicate that it is available only in few regions

Comment (toddysm) [reply docs/blog/connected-registry.md:18]: I will update the doc with additional information once we have the deployment done. For now, we just want to mitigate any questions coming from the release CLI

File: docs/blog/connected-registry.md (hunk 2 above)
Comment (dlepow) [reply docs/blog/connected-registry.md:14]: "Your premises" is OK as a noun. Adjective form should be "on-premises", per MSFT style.

File: docs/blog/connected-registry.md (hunk 3 above)
Comment (toddysm) [reply docs/blog/connected-registry.md:16]: Edge (lowercase) is an industry term so I would assume we are OK to use it. @dlepow let me know if this should be changed.

Comment (dlepow) [reply docs/blog/connected-registry.md:16]: I think(?) "edge" is OK. We use it in a lot of product names :)

File: docs/blog/connected-registry.md (hunk 2 above)
Comment (dlepow) [inline docs/blog/connected-registry.md:14]: Suggest "...synchronize images between **an Azure container registry in the cloud** and your premises..."

Not essential, but an image could also help here.

**Review by dlepow — COMMENTED** [review]
Added a few suggestions

File: docs/blog/connected-registry.md (hunk 3 above)
Comment (toddysm) [reply docs/blog/connected-registry.md:16]: Added links to the IoT Edge and Azure Stack marketing pages @SteveLasker.

File: docs/blog/connected-registry.md (hunk 2 above)
Comment (toddysm) [reply docs/blog/connected-registry.md:14]: On-premises is also industry term. It is hard writing content when we remove terms from the language :) I will change to the "on your premises" or "on premises" but I believe we should try to stick with the language the rest of the industry understands.

**Review by northtyphoon — APPROVED** [review]
LGTM

**Pull request merged**